db/db.go

```diff
@@ -0,0 +1,130 @@
+package db
+
+import (
+	"context"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+// DB database
+type DB struct {
+	pool *pgxpool.Pool
+	psql sq.StatementBuilderType
+}
+
+// New creates a new DB connection
+func New(pool *pgxpool.Pool, psql sq.StatementBuilderType) *DB {
+	return &DB{
+		pool: pool,
+		psql: psql,
+	}
+}
+
+// Post ...
```

Comment:
I know you're doing this to silence the warning, but I really recommend writing something here, even if it feels a bit pointless, to get used to doing it every tie.
```suggestion
// Post is the struct used to represent posts in JSON.
```